ext/zip/tests/oo_addpattern.phpt

```diff
@@ -24,7 +24,7 @@ if (!$zip->open($file)) {
         exit('failed');
 }
 $dir = realpath($dirname);
-$options = array('add_path' => 'baz/', 'remove_path' => $dir);
+$options = array('add_path' => 'baz', 'remove_path' => $dir);
```

Comment:
This necessary change hints at a potential BC break, so the PR should probably go to PHP 7.1 only (or maybe 8.0?). After all, this PHPT shows, that add_path and remove_path can be used together (albeit add_path needs the trailing slash, and remove_path must not have it).